server/src/test/java/com/objectcomputing/checkins/UploadControllerTest.java

```diff
@@ -120,9 +120,9 @@ void testUploadFile() throws URISyntaxException, IOException {
 
     @MockBean(GoogleDriveAccessor.class)
     public GoogleDriveAccessor googleDriveAccessor() {
-        if(googleDriveAccessor == null) {
+//         if(googleDriveAccessor == null) {
             googleDriveAccessor = mock(GoogleDriveAccessor.class);
-        }
+//         }
```

Comment:
Same with these, if they are not used and commented out, can you please remove them?